src/Platform.c

```diff
@@ -228,6 +244,22 @@ int PltCreateThread(const char* name, ThreadEntry entry, void* context, PLT_THRE
         }
         sceKernelStartThread(thread->handle, sizeof(struct thread_context), ctx);
     }
+#elif defined(__WIIU__)
+    int stack_size = 4 * 1024 * 1024;
+    void* stack_addr = (uint8_t *)memalign(8, stack_size) + stack_size;
+
+    if (!OSCreateThread(&thread->thread,
+                        (OSThreadEntryPointFn)ThreadProc,
+                        (int)ctx, NULL,
```

Comment:
Casting function pointers like this is nasty, especially since `OSThreadEntryPointFn` differs in number of parameters, parameter types, and return type compared to the `ThreadProc` you're providing. You can just define a `ThreadProc()` for `__WIIU__` like we do for Vita and avoid all those problems.

Also since you've got a pointer for the second parameter in `OSThreadEntryPointFn`, you can pass `ctx` through that rather than casting it to an int (which is not 64-bit safe, not that it really matters on Wii U).